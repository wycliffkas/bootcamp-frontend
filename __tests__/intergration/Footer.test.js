import React from "react";
import {shallow } from "enzyme";
import Footer from "../../src/components/pages/Footer";

describe("<Content />", () => {
  it("should render without crashing", () => {
    const editor = shallow(<Footer />);
    expect(editor).toMatchSnapshot();
  });
});