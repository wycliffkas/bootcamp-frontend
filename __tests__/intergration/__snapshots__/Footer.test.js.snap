// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should render without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="col-md-6 text-left"
      >
        Copyright © 2018 
        <a
          href="/"
          target="_blank"
        >
          Wycliff
        </a>
         All rights reserved.
      </div>,
      "className": "row footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "Copyright © 2018 ",
          <a
            href="/"
            target="_blank"
          >
            Wycliff
          </a>,
          " All rights reserved.",
        ],
        "className": "col-md-6 text-left",
      },
      "ref": null,
      "rendered": Array [
        "Copyright © 2018 ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Wycliff",
            "href": "/",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "Wycliff",
          "type": "a",
        },
        " All rights reserved.",
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="col-md-6 text-left"
        >
          Copyright © 2018 
          <a
            href="/"
            target="_blank"
          >
            Wycliff
          </a>
           All rights reserved.
        </div>,
        "className": "row footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Copyright © 2018 ",
            <a
              href="/"
              target="_blank"
            >
              Wycliff
            </a>,
            " All rights reserved.",
          ],
          "className": "col-md-6 text-left",
        },
        "ref": null,
        "rendered": Array [
          "Copyright © 2018 ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Wycliff",
              "href": "/",
              "target": "_blank",
            },
            "ref": null,
            "rendered": "Wycliff",
            "type": "a",
          },
          " All rights reserved.",
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
